lldb/tools/lldb-dap/Handler/InitializeRequestHandler.cpp

```diff
@@ -406,7 +406,7 @@ void InitializeRequestHandler::operator()(
   // The debug adapter supports restarting a frame.
   body.try_emplace("supportsRestartFrame", false);
   // The debug adapter supports the gotoTargetsRequest.
-  body.try_emplace("supportsGotoTargetsRequest", false);
+  body.try_emplace("supportsGotoTargetsRequest", true);
```

Comment:
You'll need to rebase this on top of #131943 and move it into the request handler.